### PR TITLE
🔊 Add sentry client to browser and node

### PR DESCRIPTION
**Pull request by jeysal**
BREAKING CHANGE: The PlaytBrowserClient now needs a `gameId` option.
Tested by linking into RR.
* Depends on
  * https://github.com/playt-net/clashparadise/pull/3027
  * https://github.com/playt-net/clashparadise/pull/3030

### CODE REVIEW

**Comment by jeysal** [comment]
@playt-net/developer Sentry finally working with this and the other PR, plz review 🙏

File: src/browser.mts [hunk 1]
```diff
@@ -1,65 +1,53 @@
...
+  gameId: string;
+  apiUrl: string;
+}) => {
   const fetcher = Fetcher.for<paths>();
   const config: FetchConfig = {
     baseUrl: apiUrl,
     use: [],
+    init: {
+      headers: {
+        'User-Agent': `playt-browser-client/${process.env.npm_package_version}`,
```
Comment (boredland) [inline src/browser.mts:21]: is this going to become static during build?

File: src/browser.mts [hunk 2]
```diff
@@ -1,65 +1,53 @@
...
   };
   fetcher.configure(config);
 
+  const initialize = async ({ gameVersion }: { gameVersion: string }) => {
+    const sentryConfigResp = await fetcher
+      .path('/api/games/{gameId}/sentry-config')
+      .method('get')
+      .create()({ gameId });
+    if (!sentryConfigResp.ok) {
+      throw new Error('Failed to fetch sentry config');
```
Comment (boredland) [inline src/browser.mts:33]: should we probably make this not an error? I wonder if this makes life unnecessarily hard for the game devs..

**Review by boredland — APPROVED** [review]
Some minor remarks.

File: src/browser.mts (hunk 1 above)
Comment (jeysal) [reply src/browser.mts:21]: yea, Babel plugin. I'm currently trying to figure out whether the headers arrive properly

Comment (jeysal) [reply src/browser.mts:21]: ok so as I already kind of feared, in the browser client `fetch`, the user-agent is ignored. backend client works

Comment (jeysal) [reply src/browser.mts:21]: I don't understand why though because the fetch spec says it's allowed to change since like 2015 or so :(

Comment (jeysal) [reply src/browser.mts:21]: Ah, it's Chromium that has still not implemented it. FF works. Well, should be good enough to get it from some browsers to see what client versions are still in usage. Also we always still get the server client version which should always be using the same one anyway. And other browsers might fix it in the future.

File: src/browser.mts [hunk 3]
```diff
@@ -1,65 +1,53 @@
...
+    init: {
+      headers: {
+        'User-Agent': `playt-browser-client/${process.env.npm_package_version}`,
+      },
+    },
   };
   fetcher.configure(config);
 
+  const initialize = async ({ gameVersion }: { gameVersion: string }) => {
+    const sentryConfigResp = await fetcher
```
Comment (jeysal) [inline src/browser.mts:28]: Firefox doesn't like the CORS here, while Chrome does o.O

Comment (jeysal) [reply src/browser.mts:28]: https://github.com/playt-net/clashparadise/pull/3030

**Pull request merged**